### ISSUE TITLE
How to get the NTHU dataset

### ISSUE THREAD

**Issue by JianghaoWu** [idiap/model-uncertainty-for-adaptation#3]
Thank you for your excellent work. I want to get the NTHU dataset mentioned in the article. Is it at 'https://yihsinchen.github.io/segmentation_adaptation_dataset/#Unlabeled'? I have not obtained the download permission so far,
Is there any other way to get the dataset?

**Comment by prabhuteja12** [comment]
Hi! 
I got it through the usual route of emailing the authors. I got it with after a few days. I don't know of any other way to get the dataset, as the authors prohibit resharing in their agreement.

**Comment by JianghaoWu** [comment]
Thank you very much for your kind reply.
I still have some questions:
1. when unc_noise is False, ie. the model is deepLab(13, False) in your code, is pretrained and it includes encoder and mian decoder right?
2. Are the parameters of the auxiliary decoder copied from the main decoder or initialized randomly?
3. When training, update the parameters of the encoder and the auxiliary decoder and keep only the parameters of the main decoder unchanged?
4. When training, the pseudo-label is generated by the pre-trained model, and it is not updated in the later training process, right?

I'd be grateful if you could answer it!
